### DISCUSSION TITLE
PowerShell for z/OS® UNIX®

### DISCUSSION THREAD

**Discussion by catherdersoftware** [zopencommunity/meta#842] [Ideas]
Earlier today during the Z Rebels Water Cooler on the SZE, I had what is either a fantastic thought or something that will get me exiled to Siberia.

PowerShell was ported to Linux environments a while ago. Even though there's a lot of "ewww! Windows!" views tainting it, I've seen several conversations on Twitter about how some Linux sysadmins love it, and it provides some common tooling for those sysadmins who have to deal with both.

Enter z/OS® UNIX®. We are trying to make administration easier for those who grew up outside of z/OS. Would it be useful to port a PowerShell subset to USS, and add some cmdlets for obvious things? 

Let the flames begin. (Are flame wars still a thing? Yes, I'm a child of the early 1990s Internet.)

**Comment by v1gnesh** [comment]
[PowerShell is written in C#](https://github.com/PowerShell/PowerShell).
z/OS doesn't have C#, but LoZ does.
